File: java/HelloWorld.java
public class HelloWorld {
  public String texto;

  public String acessarTexto() {
    return this.texto;
  }

  public void modificarTexto(String texto) {
    this.texto = texto;
  }

  public void imprimirTexto() {
    System.out.println(this.acessarTexto());
  }

  public static void main(String[] args){
    HelloWorld helloWorld = new HelloWorld();
    helloWorld.modificarTexto("Hello, World!");
    helloWorld.imprimirTexto();
  }
}

File: python/hello-world.py
class HelloWorld:
    def __init__(self, texto):
        self.texto = texto
    
    def imprimir_texto(self):
        print(self.texto)

def main():
    hello_world = HelloWorld("Hello, World!")
    hello_world.imprimir_texto()

if __name__ == '__main__':
    main()
